src/components/NewsRow.tsx

```diff
@@ -0,0 +1,67 @@
+import djs from "dayjs";
+import { useNavigate } from "react-router-dom";
+
+import { MarketNewsQuery } from "../types";
+
+interface NewsRowProps {
+  title: MarketNewsQuery["marketNews"][number]["title"];
+  url: MarketNewsQuery["marketNews"][number]["url"];
+  time: MarketNewsQuery["marketNews"][number]["time"];
+  publisher: MarketNewsQuery["marketNews"][number]["publisher"];
+  imageUrl?: MarketNewsQuery["marketNews"][number]["imageUrl"];
+  subjects: MarketNewsQuery["marketNews"][number]["subjects"];
+}
+
+const NewsRow = ({ title, url, time, publisher, imageUrl, subjects }: NewsRowProps): JSX.Element => {
+  let navigate = useNavigate();
+  return (
+    <div className="border-b border-y-2 sm:w-full md:w-9/12 py-3 flex flex-col gap-3">
+      <a href={url} target="_blank">
+        <tr className="w-full flex flex-row items-center hover:bg-slate-100 hover:cursor-pointer">
```

Comment:
I don't _think_ you need the `tr`/`td` tags in this element, since the parent is not a `table`. Also I get some browser warnings in chrome about nesting other elements inside

Reply:
replaced with divs, my bad on that one